azure-pipelines.yml

```diff
@@ -29,80 +30,97 @@ pr:
     include:
     - '*'
 
-jobs:
-- template: /eng/common/templates/jobs/jobs.yml
-  parameters:
-    enableMicrobuild: true
-    enablePublishBuildArtifacts: true
-    enablePublishTestResults: true
-    enablePublishBuildAssets: true
-    enablePublishUsingPipelines: $(_PublishUsingPipelines)
-    enableTelemetry: true
-    helixRepo: aspnet/Blazor
-    jobs:
-    - job: Windows
-      pool:
-        ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          vmImage: windows-2019
-        ${{ if ne(variables['System.TeamProject'], 'public') }}:
-          name: NetCoreInternal-Pool
-          queue: BuildPool.Windows.10.Amd64.VS2019
-      variables:
-        - name: _BuildConfig
-          value: Release
-        - name: _BuildArgs
-          value: ''
-        # PRs or external builds are not signed.
-        - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-          - name: _SignType
-            value: test
-          - name: _DotNetPublishToBlobFeed
-            value: false
-        # Only enable publishing in official builds.
-        - ${{ if and(eq(variables['System.TeamProject'], 'internal'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-          # DotNet-Blob-Feed provides: dotnetfeed-storage-access-key-1
-          # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
-          - group: DotNet-Blob-Feed
-          - group: Publish-Build-Assets
-          - name: _SignType
-            value: real
-          - name: _DotNetPublishToBlobFeed
-            value: true
+stages:
+- stage: build
+  displayName: Build
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      enablePublishBuildArtifacts: true
+      enablePublishTestResults: true
+      enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
+      enableTelemetry: true
+      helixRepo: aspnet/Blazor
+      ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        enableMicrobuild: true
+        enablePublishBuildAssets: true

```

Comment:
Unrelated bug. Should not publish build assets or have signing machinery in place in external builds. But, I was slightly surprised test signing works without Microbuild 😈